app/Providers/AppServiceProvider.php

```diff
@@ -55,6 +62,94 @@ public function boot()
         $this->bootObservers();
     }
 
+    private function bootFlare(): void
+    {
+
+        // PHPStan (Larastan?) prevents us from skipping loading the service provider at all
+        // Not sure if bug or by design
+        if ($this->app->isProduction() && \LibreNMS\Config::get('reporting.error', false) == true) {
+            Config::set('flare.key', Config::get('librenms.flare_key'));
+        }
+
+        $this->app->register(\Facade\Ignition\IgnitionServiceProvider::class);
+
+        /**
+         * Filter reports based on conditions
+         */
+        Flare::filterReportsUsing(function (Report $report) {
+            // Only run in production
+            if (! $this->app->isProduction()) {
+                return false;
+            }
+
+            // Check if git installation
+            if (! Git::repoPresent()) {
+                return false;
+            }
+
+            // Repo url must be offical one
+            if (! Str::contains(Git::remoteUrl(), ['git@github.com:librenms/librenms.git', 'https://github.com/librenms/librenms.git'])) {
```

Comment:
This check is wrong since it only checks origin branch, and not *current* branch.